x/dsmr/node_test.go

```diff
@@ -1378,3 +1512,38 @@ func newTestNodes(t *testing.T, n int) []*Node[dsmrtest.Tx] {
 
 	return result
 }
+
+type testingTimeValidityWindowMock struct {
+	OnIsRepeat func(ctx context.Context,
+		parentBlk validitywindow.ExecutionBlock[*emapChunkCertificate],
+		txs []*emapChunkCertificate,
+		oldestAllowed int64,
+	) (set.Bits, error)
+
+	OnVerifyExpiryReplayProtection func(context.Context, validitywindow.ExecutionBlock[*emapChunkCertificate], int64) error
+
+	defaultIsRepeatErr                     error
+	defaultVerifyExpiryReplayProtectionErr error
```

Comment:
Can we just use the functions and make this a no-op that returns a nil error if they are not populated?